tests/test_orm_mode.py

```diff
@@ -357,3 +357,47 @@ class Config:
 
     # Pass dictionary data directly
     State(**{'user': {'first_name': 'John', 'last_name': 'Appleseed'}})
+
+
+def test_orm_mode_with_kwargs():
+    class User(BaseModel):
+        username: str
+        name: str
+
+        class Config:
+            orm_mode = True
+
+    class UserCls:
+        username: str
+        full_name: str
+
+        def __init__(self, username: str, full_name: str) -> None:
+            self.username = username
+            self.full_name = full_name
+
+    user = UserCls('admin', 'john smith')
+
+    converted = User.from_orm(user, name=user.full_name)
+
+    assert user.username == converted.username
+    assert user.full_name == converted.name
```

Comment:
This test is pretty confusing, better to take the logic from the docs example and strip out the sqlalchemy dependency.